### PR TITLE
TS errors to 0

### PR DESCRIPTION
**Pull request by Kadrian**

### CODE REVIEW

File: frontend/src/js/concept-trees/ConceptTreeNode.tsx [hunk 1]
```diff
@@ -107,7 +107,9 @@ const ConceptTreeNode: FC<PropsT> = ({
             ids: [conceptId],
             ...description,
             label: data.label,
-            tables: resetTables(concept?.tables, { useDefaults: true }),
+            tables: concept?.tables
+              ? resetTables(concept.tables, { useDefaults: true })
+              : [],
             selects: resetSelects(concept?.selects, { useDefaults: true }),
```
Comment (awildturtok) [inline frontend/src/js/concept-trees/ConceptTreeNode.tsx:113]: for resetTables you are handling null outside, but I guess in resetSelects you are handling it inside? Would make more sense to go for one or the other approaches.

File: frontend/src/js/concept-trees/globalTreeStoreHelper.ts [hunk 2]
```diff
@@ -143,7 +141,9 @@ export const getConceptsByIdsWithTablesAndSelects = (
   return {
     concepts: conceptIds.map((id) => getConceptById(id)).filter(exists),
     root: rootConceptId,
-    tables: resetTables(rootConcept.tables, resetConfig),
+    tables: rootConcept.tables
+      ? resetTables(rootConcept.tables, resetConfig)
+      : [],
```
Comment (awildturtok) [inline frontend/src/js/concept-trees/globalTreeStoreHelper.ts:146]: Same as above

**Review by awildturtok — COMMENTED** [review]
Very nice!

File: frontend/src/js/concept-trees/ConceptTreeNode.tsx (hunk 1 above)
Comment (Kadrian) [reply frontend/src/js/concept-trees/ConceptTreeNode.tsx]: good point, fixed that, handling it always outside now

**Pull request merged**